.github/workflows/check-change.yml

```diff
@@ -0,0 +1,17 @@
+name: Check change
+on:
+  pull_request:
+
+jobs:
+  check-change:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14'
+
+      - run: |
+          cp scripts/beachball/.beachballrc.base.json .beachballrc.json
+          npx beachball@2.18.0 check --scope "!packages/fluentui/*" --changehint "Run 'yarn change' to generate a change file"
```

Comment:
`npm run checkchange`

Reply:
Replaced with `yarn checkchange` as we have `yarn` in the image and it's better to use the same thing as we force locally.